### PR TITLE
Fix for #13711

### PR DESCRIPTION
**Pull request by Dartui**
I think this is the best way to fix this issue. It does not make any breaking changes in code and is BC, because it is not modifying email trigger.

### CODE REVIEW

File: includes/admin/meta-boxes/class-wc-meta-box-order-actions.php [hunk 1]
```diff
@@ -130,7 +130,13 @@ public static function save( $post_id, $post ) {
 				if ( ! empty( $mails ) ) {
 					foreach ( $mails as $mail ) {
 						if ( $mail->id == $email_to_send ) {
-							$mail->trigger( $order->get_id(), $order );
+							if ( 'customer_refunded_order' === $email_to_send ) {
+								$partial_refund = $order->get_remaining_refund_amount() != 0 || $order->get_remaining_refund_items() != 0;
```
Comment (mikejolley) [inline includes/admin/meta-boxes/class-wc-meta-box-order-actions.php]: > or < checks would be better here.

**Comment by mikejolley** [comment]
@Dartui Why is that email even in there to resend? Thats not one which should IMO because not all orders **have** refunds. Sending the invoice again shows refunds.

**Comment by Dartui** [comment]
@mikejolley I don't know how it looks in UK, but in Poland invoice is a document with order summary generated after payment, mostly in PDF. For now in WooCommerce, the best way to send this document to customer is attach it to Order Completed mail. Invoice mail is useless, because it has to be manually sent.
Refund email can be used for sending automatic generated correction to invoices. This is how I am using it, because when I make a refund I have to create correction. It should not be removed, but can be hidden when order has no refunds.

**Comment by mikejolley** [comment]
@Dartui When you make a refund, it sends an email on it's own. No need to trigger.

**Comment by Dartui** [comment]
@mikejolley what if for some reason customer do not receive refund email? Each auto generated order email has resend option, for me this should stay as is.

**Comment by mikejolley** [comment]
Then you should fix that because it's triggered :P It doesn't make sense to show this one because it can ONLY apply to orders which have been refunded. Otherwise it is invalid and really odd.

**Comment by Dartui** [comment]
@mikejolley I have added conditional for showing this option based on order refunds count. Also changed comparison operator. What you think about this?

**Comment by mikejolley** [comment]
I still don't think it makes sense because that email is for a specific refund event. If you resend it, the user is going to assume they have been refunded *again*. I think it's misleading.

@claudiulodro @justinshreve thoughts?

**Comment by justinshreve** [comment]
@mikejolley I think there is a valid use-case in needing to resend a message - stuff happens. The original send could fail, or the user could claim they never got notified.

> If you resend it, the user is going to assume they have been refunded again. I think it's misleading.

If an email is triggered this way, could we include something in the email that says this was for a previous refund (i.e. for a refund of $xx on x/x/xx) and is just a duplicate notification/resend?

**Comment by mikejolley** [comment]
@justinshreve The problem is it's event based. What do you send if you have multiple refunds in an order?

**Comment by claudiulodro** [comment]
I'm not sure what the best solution to this problem is. It is definitely important that owners are able to get information to people that missed an e-mail. I don't necessarily have a big problem with keeping "Resend refunded order", but it is a little confusing because it doesn't specify which refund, can be used on orders that don't have refunds, and customers may think they got refunded twice if they already got a refund email. 

IMO the way to go would be to have a "Send order details to customer" option that could give all of the order details, status, and total refunded amount (if refunds happened). If someone has questions about anything in their order or missed an e-mail, just send them that.

**Comment by mikejolley** [comment]
> IMO the way to go would be to have a "Send order details to customer" option that could give all of the order details, status, and total refunded amount (if refunds happened). If someone has questions about anything in their order or missed an e-mail, just send them that.

Thats what invoice does AFAIK

**Comment by Dartui** [comment]
Both refund and invoice email looks the same. They are displaying each refund, even emails triggered by refund event. The only differences are subject and headings.

**Comment by claudiulodro** [comment]
That's what I thought. :) Since the invoice e-mail has all of the information the resent refund e-mail does, I don't think it makes much sense to keep the "Resend refunded order" option in the admin, since it is more confusing than helpful.

**Comment by mikejolley** [comment]
Maybe we need a generic "resend order details" that can be a catch all for these emails. I'd propose we introduce that in 3.1.0, and for this release, just hide the broken refund email from view.

**Comment by mikejolley** [comment]
Logged here https://github.com/woocommerce/woocommerce/issues/13738

**Pull request closed without merging**